.github/workflows/atd_moped_build_stage.yml

```diff
@@ -90,40 +90,6 @@ jobs:
           source $(pwd)/.github/workflows/aws-heroku-helper.sh
           build_database
           run_database_migration
-  build_activity_log_sqs:
```

Comment:
Wondering if we should remove this file entirely. This is the github action to trigger the old moped test deployment. It will not work because the heroku script was removed a while back https://github.com/cityofaustin/atd-moped/pull/781

Reply:
if we're not using any of this—yes! ➡️ 🗑️

Reply:
Agreed - 👋 🗑️